fonts/squarize.py

```diff
@@ -379,913 +433,991 @@ def end_glyph(glyphnumber):
+    elif (first_glyph == "pbase" or first_glyph == "p2base"):
+        width_difference = WIDTH_PUNCTUM-WIDTH_HIGH_PES
     else:
-        width_difference=0
-    if (width_difference<0):
+        width_difference = 0
+    if (width_difference < 0):
         paste_and_move(first_glyph, glyphnumber, -width_difference, 0)
     else:
         simple_paste(first_glyph, glyphnumber)
-    if (first_glyph=="qbase"):
-        width1=width_quilisma-line_width
+    if (first_glyph == "qbase"):
+        width1 = WIDTH_QUILISMA-LINE_WIDTH
     else:
-        width1=width_punctum-line_width
-    if (width_difference<0):
-        width1=width1-width_difference
-    write_line(i, glyphnumber, width1, base_height)
+        width1 = WIDTH_PUNCTUM-LINE_WIDTH
+    if (width_difference < 0):
+        width1 = width1-width_difference
+    write_line(i, glyphnumber, width1, BASE_HEIGHT)
     if (width_difference != 0):
-        paste_and_move("phigh", glyphnumber, width_difference, i*base_height)
+        paste_and_move('phigh', glyphnumber, width_difference, i*BASE_HEIGHT)
     else:
-        paste_and_move("phigh", glyphnumber, 0, i*base_height)
-    if (width_difference<0):
-        set_width(glyphnumber, width_punctum)
+        paste_and_move('phigh', glyphnumber, 0, i*BASE_HEIGHT)
+    if (width_difference < 0):
+        set_width(glyphnumber, WIDTH_PUNCTUM)
     else:
-        set_width(glyphnumber, width_quilisma)
-    end_glyph(glyphnumber)    
+        set_width(glyphnumber, WIDTH_QUILISMA)
+    end_glyph(glyphnumber)
 
 def write_pes_debilis(i, shape, liquescentia='nothing'):
-    glyphnumber=gnumber(i, 0, 0, shape, liquescentia)
+    glyphnumber = gnumber(i, 0, 0, shape, liquescentia)
     # with a deminutus it is much more beautiful than with a idebilis
     paste_and_move("deminutus", glyphnumber, 0, 0)
-    write_line(i, glyphnumber, width_debilis, base_height)
+    write_line(i, glyphnumber, WIDTH_DEBILIS, BASE_HEIGHT)
     simplify(glyphnumber)
-    paste_and_move("base4", glyphnumber, width_debilis, i*base_height)
-    set_width(glyphnumber, width_punctum+width_debilis)
-    end_glyph(glyphnumber) 
+    paste_and_move("base4", glyphnumber, WIDTH_DEBILIS, i*BASE_HEIGHT)
+    set_width(glyphnumber, WIDTH_PUNCTUM+WIDTH_DEBILIS)
+    end_glyph(glyphnumber)
 
 def write_pes_deminutus(i, first_glyph, shape, liquescentia='nothing'):
-    glyphnumber=gnumber(i, 0, 0, shape, liquescentia)
+    glyphnumber = gnumber(i, 0, 0, shape, liquescentia)
     simple_paste(first_glyph, glyphnumber)
-    if (first_glyph=="qbase"):
-        width1=width_quilisma-line_width
+    if (first_glyph == "qbase"):
+        width1 = WIDTH_QUILISMA-LINE_WIDTH
     else:
-        width1=width_punctum-line_width
-    write_line(i, glyphnumber, width_punctum-line_width, base_height)
-    paste_and_move("rdeminutus", glyphnumber, width_punctum-line_width-width_deminutus, i*base_height)
-    set_width(glyphnumber, width_punctum)
+        width1 = WIDTH_PUNCTUM-LINE_WIDTH
```

Comment:
This `if` statement is currently useless. Should the next line (`590`) use `width1` rather than `WIDTH_PUNCTUM_LINE_WIDTH` ? This would be consistent with line `562`.